.github/workflows/diff-change-to-dist.yaml

```diff
@@ -23,22 +23,21 @@ jobs:
       - name: Generate diff
         id: diff
         run: |
-          diff=$(git diff -M1 origin/$GITHUB_BASE_REF -- dist)
-          # Escape new lines
-          diff="${diff//'%'/'%25'}"
-          diff="${diff//$'\n'/'%0A'}"
-          diff="${diff//$'\r'/'%0D'}"
-          echo "::set-output name=diff::$diff"
+          git diff -M1 origin/$GITHUB_BASE_REF -- dist \
```

Comment:
I think in the last release we also had an issue because the similarity was too low, so it saw the change as a deletion and addition rather than a rename.

Looking at my history, it looks like when I generated the diff manually I had to use `-M05`, so I'm wondering if we should make that change here too?

Reply:
Oh yes, I remember that too, I will add that change in an additional commit.